File: miband-sdk/src/main/java/com/khmelenko/lab/miband/listeners/HeartRateNotifyListener.java
package com.khmelenko.lab.miband.listeners;

/**
 * Listener for hear rate notifications
 *
 * @author Dmytro Khmelenko
 */
public interface HeartRateNotifyListener {

    /**
     * Called when new hear rate data received
     *
     * @param heartRate Hear rate
     */
    void onNotify(int heartRate);
}

File: miband-sdk-kotlin/src/main/java/com/khmelenko/lab/miband/listeners/HeartRateNotifyListener.kt
package com.khmelenko.lab.miband.listeners

/**
 * Listener for hear rate notifications
 *
 * @author Dmytro Khmelenko
 */
interface HeartRateNotifyListener {

    /**
     * Called when new hear rate data received
     *
     * @param heartRate Hear rate
     */
    fun onNotify(heartRate: Int)
}
